lambda.tf

```diff
@@ -0,0 +1,63 @@
+locals {
```

Comment:
I suggest renaming this file into `autoscaler` and having everything related in there.

Reply:
Done